src/pages/home/Home.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/no-array-index-key */
+import React, { useEffect, useReducer } from 'react';
+import { RowSection, Banner } from '../../components';
+import { getProducts } from '../../services/flixycartApi';
+import { productReducer } from './reducers/productReducer';
+import { loadProductActions } from '../../constants/actions';
+import './Home.scss';
+
+const Home = () => {
+  const [productState, dispatch] = useReducer(productReducer, { loading: true, product: [] });
+
+  useEffect(() => {
+    document.title = 'Flixycart | Your store for latest books';
+
+    dispatch({ type: loadProductActions.REQUEST_GET_PRODUCTS });
+
+    getProducts()
+      .then((res) => {
+        dispatch({
+          type: loadProductActions.REQUEST_GET_PRODUCTS_SUCCESS,
+          payload: res.data.result,
+        });
```

Comment:
And why to store this Actions in objects. 
is it a good practice?
By this way you are writing extra code.

You could have simply passed the Action as the string?

Reply:
Yes, it's the recommended practice. Since, actions are strings, and there are high chances of mistakes working directly with strings. Chances of missing a word or replacing a letter with another. 
Ex:
In case of ```REQUEST_GET_PRODUCTS_SUCCESS```, if I miss a word such as ```REQUEST_GET_PRODUCTS_SUCCESSS```(added extra 'S' at the end).. This will break my app.
Hence, to avoid such things, one should always keep his actions in some centralized actions object and import them.

Reply:
Hey Thanks @Debarshi95 for letting me know that.